scripts/components/inline-notification/docs/story.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import readme from './readme.mdx';
+import { InlineNotification, InlineNotificationType } from '../inline-notification';
+
+export default {
+	title: 'Options/InlineNotification',
+	parameters: {
+		docs: {
+			page: readme
+		}
+	},
+};
+export const regular = () => {
+	return (
+		<div style={{
+			display: 'flex',
+			flexDirection: 'column',
+			gap: '1rem',
+		}}>
+			<InlineNotification
+				text='This is a warning!'
```

Comment:
Translations.

Reply:
storybook doesn't need translations it is dummy data